ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -61,10 +61,36 @@ jobs:
     requires: [fedora-latest/build]
     priority: 50
     job:
-      class: RunPytest
+      class: RunWebuiTests
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_webui/test_automount.py
         template: *ci-master-latest
         timeout: 3600
-        topology: *master_1repl_1client
+        topology: *ipaserver
+
+  fedora-rawhide/build:
+    requires: []
+    priority: 100
+    job:
+      class: Build
+      args:
+        git_repo: '{git_repo}'
+        git_refspec: '{git_refspec}'
+        template: &ci-master-frawhide
+          name: freeipa/ci-master-frawhide
+          version: 0.0.10
+        timeout: 1800
+        topology: *build
+
+  fedora-rawhide/temp_commit:
+    requires: [fedora-rawhide/build]
+    priority: 50
+    job:
+      class: RunWebuiTests
+      args:
+        build_url: '{fedora-frawhide/build_url}'
```

Comment:
Remove the `f`:
```
build_url: '{fedora-frawhide/build_url}
--------------------^
```
Also, you need to rebase because of https://github.com/freeipa/freeipa/pull/4108/files#diff-46f33c2694f0a52173ef4722db670b48